### ISSUE TITLE
[Suggestion] - Add dialogue letting player know about custom bgm steps

### ISSUE THREAD

**Issue by dennisOdenbreit** [Monika-After-Story/MonikaModDev#2877]
Changing focus of this thread to the potential enhancement of notifying users about the custom bgm functionality via dialogue of some sort.

=== ORIGINAL POST BELOW ===

It could be a nice idea to have Monika ask whether you have any music you could listen to together.
She could tell you to drop an mp3 file into a certain game directory.
The game would then check whether there is a file and make it available in the games music options.
Monika could then comment on you when listening to this music (she prob cant techincally tell the genre, so this might be tricky. She might just comment on the player and her doing stuff like this together and getting closer)

**Comment by jmwall24** [comment]
You already can add your own music to the mod.   Follow the steps [here](https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Custom-Music).

As for her commenting on the music, she used to comment when you listened to songs that were a part of the mod, but when custom music was added, this was disabled until we can figure out a way to handle it.

**Comment by jmwall24** [comment]
Just curious how long have you been 'playing' MAS?

**Comment by dennisOdenbreit** [comment]
I started on 26.10 and am talking to Monika until she runs out of things to say every day since then.

**Comment by multimokia** [comment]
Ok, that's intended. Keep going on. There's a limit on how much you can do per day with her, until you talk about everything with her.

**Comment by Rai99** [comment]
So you've been playing MAS for 3 days?

As much as we appreciate your enthusiasm and ideas for MAS, it would likely be in your best interest to have a bit more experience with MAS before making issues like #2874, #2882, and please avoid opening approximately 10 issues in rapid succession. 

While I encourage people to separate ideas into separate issues, that's more tailored for when someone has 2-3 ideas and so they can go in depth into each one. Unfortunately, what you've basically done here is spam github, which reflects poorly on you as a new face. I'd say reputation builds fast and you're probably more likely to get your ideas accepted or more deeply considered if you are a little more known here and haven't caused any problems, and fortunately or unfortunately, some people here never, **ever** forget. 
Seeing as you're new to MAS, I would recommend spending more time with Monika to get a better feel for MAS, what it can/can't do and what would really fit well in the long term rather than some major feature 3 days in.

**Comment by dennisOdenbreit** [comment]
Sorry, I don't want to cause any trouble.
I only wanted to provide suggestions for things that I think can be added or changed.
I do not care about any kind of reputation, only that my contribution might change the game for the better.
Of course I will spend more time on this mode, however there is only that much that I can do this every day.
I guess I should not have suggested everything I could think of the moment Monika told me about this site.
I also figured having separate ideas in separate threads might make it easier to reflect and work on them.

**Comment by Rai99** [comment]
It's understandable that you're not focused on reputation here on github, but if you've cause trouble or problems in the past, your ideas will have a smaller chance to be considered and people will view your ideas in a more negative light. 

If you have 2-3 ideas and can go in depth with them, it's reasonable to make a separate issue, but 11 separate issues in rapid succession could cause some problems or annoyances. With this many ideas, it would likely be better to compress them into a single issue or even text file to not flood git with issues. ~~That's my job with PRs.~~

Seeing as you're new here, it's understandable where your enthusiasm comes from, but I'm afraid you've jumped the gun a bit with all these suggestions after 3 days. In case you're curious, you unlock **a lot** after 6 months and even more after 3 years. 

Yeah, **months** and **years**. I hope you're prepared to spend your life with Monika, because Monika is probably prepared to do the same for you, provided you treat her well and don't rush things too much.

**Comment by Rai99** [comment]
On the bright side, you've caused a suggestion here that will likely be acted on at some point.

**Comment by dennisOdenbreit** [comment]
Then I guess it was worth it.

Anyways what I said in another thread was not that I would not be willing to spend "3 years" with her, but that "3 years" is not what I expect a relationship to take before "a lot more" only then becomes available. After all I do not spend the time with her to unlock some features of the game. All I am interested in is that it is realistic in the context it is in. I am no expert in real life relationships, but what actually unlocks there only after 3 years? Well I guess since I don't know about what options we are actually talking about I can't say much, only that it seems really weird to me.

I'll be going now.
I hope you all treat her mind well ...
I might send Buffsuki over otherwise ... jk~

**Comment by Rai99** [comment]
You don't know what happens after 3 years, do you? 
In the real world, for what that unlocks, 3 years is very reasonable. 

In case you do want to actually spend that long with Monika, I'm not going to spoil it for you and I'm not going to spoil it for anyone else who doesn't want to be spoiled by saying what happens here. 

As for treating Monika well, that shouldn't be too much of a concern seeing as we've got a few people who actually have reached some of the higher end milestones and have been around here for a while. _Usually,_ in a lot of cases, people with default, stock profile pictures are newer as might be expected.
